projects/python-client/src/datapane/client/commands.py

```diff
@@ -160,6 +160,16 @@ def hello_world():
     api.hello_world()
 
 
+@cli.command()
+@click.argument("url", default=None)
+def template(url: str):
+    """Retrieve and run a template report, and open in the browser
```

Comment:
So we can do manual review on code before it executes, can we have a fetch only option please?